services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -642,6 +642,68 @@ class FlowCrudSpec extends BaseSpecification {
         Wrappers.wait(WAIT_OFFSET) { assert !islUtils.getIslInfo(newIsl).isPresent() }
     }
 
+    def "Systems allows to pass traffic via full and vlan port when they are on the same port"() {
+        when: "Create a flow with vlan port"
+        def (Switch srcSwitch, Switch dstSwitch) = topology.activeTraffGens*.switchConnected
+        def bandwidth = 100
+        def flowVlanPort = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        flowVlanPort.maximumBandwidth = bandwidth
+        flowVlanPort.allocateProtectedPath = true
+        flowHelper.addFlow(flowVlanPort)
+
+        and: "Create the same flow with full port"
+        def flowFullPort = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        flowFullPort.maximumBandwidth = bandwidth
+        flowFullPort.source.vlanId = 0
+        flowFullPort.destination.vlanId = 0
+        flowFullPort.allocateProtectedPath = true
+        flowHelper.addFlow(flowFullPort)
+
+        then: "The flow with full port has less priority than flow with vlan port"
+        def flowVlanPortInfo = database.getFlow(flowVlanPort.id)
+        def flowFullPortInfo = database.getFlow(flowFullPort.id)
+
+        def rules = [srcSwitch.dpId, dstSwitch.dpId].collectEntries {
+            [(it): northbound.getSwitchRules(it).flowEntries]
+        }
+
+        // can't be imported safely org.openkilda.floodlight.switchmanager.SwitchManager.FLOW_FULL_PORT_PRIORITY
+        // [ERROR] General error during semantic analysis: java.lang.NoClassDefFoundError:
```

Comment:
the info is not valuable. We know why that happens, why insert error text?

Reply:
text of error was removed